package.json

```diff
@@ -37,14 +37,17 @@
     "start:playground": "npm run storybook",
     "std-version": "standard-version -m \"chore(release): version %s build ${TRAVIS_BUILD_NUMBER} [ci skip]\"",
     "storybook": "start-storybook -p 12123",
+    "storybook:ci": "start-storybook -p 12123 --ci --quiet",
     "style:lint": "stylelint 'src/**/*.scss' 'src/**/*.css'",
     "test:coverage:watch": "npm run test:coverage -- --watch",
     "test:coverage": "npm run test:dev -- --coverage",
     "test:coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js || echo -e \"Coveralls failed.\"",
     "test:dev": "jest --config=./config/jest/jest.config.js",
-    "test:watch": "npm run test:dev -- --watch",
-    "test": "npm run test:watch",
-    "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive"
+    "test": "npm run test:dev -- --watch",
+    "test:visual:dev": "BABEL_ENV=test jest --config=./visual-regression-testing/jest.config.js",
+    "test:visual:update:dev": "npm run test:visual:dev -- --updateSnapshot",
+    "test:visual": "server-test storybook:ci http-get://localhost:12123 test:visual:dev",
+    "test:visual:update": "server-test storybook:ci http-get://localhost:12123 test:visual:update:dev"
```

Comment:
So it's just reusing the ci script so that storybook doesn't launch. It's meant to be manual and isn't in the travis config.

Reply:
Sorry, when should `test:visual:update:dev` be run then?

Reply:
`test:visual:update:dev` is run inside `test:visual:update` - storybook needs to be running for it to be able to take screenshots

Reply:
this is a 2 minute read that will clarify why: https://storybook.js.org/docs/testing/automated-visual-testing/

Reply:
I did read that; to me, it's getting confusing which scripts to use in a regular workflow and which are for convenience of CI or another script.

Since it's not called anywhere else, what are your thoughts on nixing `test:visual:update:dev` entirely and writing it inline? Or is there value in keeping both?
```
"test:visual:update": "server-test storybook:ci http-get://localhost:12123 test:visual:dev -- --updateSnapshot"
```

_If I'm the only one, feel free to ignore._

Reply:
It would be great, but that's not how `server-test` works. You can't pass the argument like that in here - it doesn't work. Feel free to try, maybe I did it wrong 🤷‍♀

Reply:
I see, so it's a CLI limitation. It looks like it allows shorthand aliases for npm scripts, but you can also use the whole command surrounded by quotes. 

```
"test:visual:update": "server-test storybook:ci http-get://localhost:12123 'npm run test:visual:dev -- --updateSnapshot'"
```

I'll leave it up to you if you think it's valuable enough to change; I just know too many npm scripts can actually make things **more** confusing when reading over them later.

Reply:
Cool! I didn't know you could do it that way. Feel free to push a change if you want it, I don't care either way.